source/simulator/assemblers/newton_stokes.cc

```diff
@@ -519,7 +552,7 @@ namespace aspect
 
       Assert(!this->get_parameters().enable_elasticity
              ||
-             outputs.template get_additional_output<MaterialModel::ElasticOutputs<dim>>()->elastic_force.size()
```

Comment:
This still needs to be true, right? The elastic_force is used in line 370 (new version)